event.json

```diff
@@ -0,0 +1,14 @@
+{
+  "pull_request": {
+    "number": 1,
+    "title": "Test PR",
```

Comment:
The title of the pull request should be more descriptive to convey the purpose of the changes made.